### PR TITLE
Add MSSQL demo

### PR DESCRIPTION
**Pull request by dymurray**

### CODE REVIEW

File: demos/2019_07_Hackfest/Setup.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+```
+
+1. Determine the route of `mig-ui`, we will use this later to enable a CORS header on the OCP 3.x side.
+```bash
+$ oc get routes migration -n mig -o jsonpath='{.spec.host}'
+migration-mig.apps.cluster-dymurray-ocp4.dymurray-ocp4.mg.example.com
+```
+### Enable CORS Headers on Source and Destination
+
+1. Run the following playbook to enable the proper CORS headers on both the source and destination clusters
```
Comment (pranavgaikwad) [inline demos/2019_07_Hackfest/Setup.md]: this will only enable CORS on source env (3.x), the destination CORS settings are taken care of by the operator

Comment (dymurray) [reply demos/2019_07_Hackfest/Setup.md]: Thank you

File: demos/2019_07_Hackfest/Setup.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+
+### Enable CORS Headers on Source cluster
+
+1. Run the following playbook to enable the proper CORS headers on the source cluster
+```bash
+$ cd ../cors/
+$ ansible-playbook cors.yaml
+```
+
+## Prepare to use mig-ui from OCP 4.1 Cluster in your Browser
```
Comment (jwmatthews) [inline demos/2019_07_Hackfest/Setup.md]: I might have missed this, but we should call out need to accept the cert for the source cluster as well.
When we go into the Migration Plan wizard and do namespace discovery, browser will error out if we haven't accepted the source clusters cert prior.

Having user visit the console on source side and accept the cert I think is sufficient.

File: demos/2019_07_Hackfest/Setup.md [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+1. Verify that the `migration-operator` deployment is running in `mig` namespace and healthy
+```bash
+$ oc get deployment migration-operator  -n mig
+NAME                 DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+migration-operator   1         1         1            1           5m
+```
+
+1. Verify that Velero is now running on the source in the `mig` namespace
+```
+$ oc get pods -n mig | grep velero
```
Comment (pranavgaikwad) [inline demos/2019_07_Hackfest/Setup.md]: best to add a note about Restic too

**Pull request merged**